src/components/article-preview.js

```diff
@@ -1,20 +1,40 @@
 import React from 'react'
 import { Link } from 'gatsby'
-import Img from 'gatsby-image'
+import { GatsbyImage } from 'gatsby-plugin-image'
 
-import styles from './article-preview.module.css'
+import Container from './container'
+import Tags from './tags'
+import * as styles from './article-preview.module.css'
 
-export default ({ article }) => (
-  <div className={styles.preview}>
-    <Img alt="" fluid={article.heroImage.fluid} />
-    <h3 className={styles.previewTitle}>
-      <Link to={`/blog/${article.slug}`}>{article.title}</Link>
-    </h3>
-    <small>{article.publishDate}</small>
-    <p
-      dangerouslySetInnerHTML={{
-        __html: article.description.childMarkdownRemark.html,
-      }}
-    />
-  </div>
-)
+const ArticlePreview = ({ posts }) => {
+  if (!posts) return null
+  if (!Array.isArray(posts)) return null
+
+  return (
+    <Container>
+      <ul className={styles.articleList}>
+        {posts.map((post) => {
+          return (
+            <li key={post.slug}>
+              <Link to={`/blog/${post.slug}`} className={styles.link}>
```

Comment:
How strongly do you feel about having the entire card contents be inside of this link? Asking bc when using a screen reader, it'll read out the entire contents as the link text. It also doesn't include punctuation between things like the title, description, date, and tags, so it just reads them all out in one breath.

We could make just the h2 a link instead, so that the rest of the elements will be read out separately

![Screen Shot 2021-08-04 at 11 12 56 AM](https://user-images.githubusercontent.com/3609989/128232842-0502a9c4-a7a9-410e-a370-3dd4eee5f3c1.png)

Reply:
I changed things from exactly what you suggest to this, and am hereby calling out my own ignorance. Thank you, Megan – will adjust! :v: 🙏

Reply:
Adjusted to just link image and title in e3bb4e3.